### PR TITLE
Show version on REPL startup

### PR DESCRIPTION
**Pull request by toshihidetagami**
This pull request closes issue #7.

### CODE REVIEW

File: src/main/scala/com/kubukoz/next/Main.scala [hunk 1]
```diff
@@ -87,7 +88,7 @@ object Main extends CommandIOApp(name = "spotify-next", header = "spotify-next:
     def reportError(e: Throwable): IO[Unit] =
       Console[IO].errorln("Command failed with exception: ") *> IO(e.printStackTrace())
 
-    fs2.Stream.exec(IO.println("Loading REPL...")) ++
+    fs2.Stream.exec(IO.println(s"Loading REPL...version: ${BuildInfo.version.takeWhile(_ != '+')}")) ++
```
Comment (kubukoz) [inline src/main/scala/com/kubukoz/next/Main.scala:91]: IMO we can print the full version. There isn't going to be a `+` in the tagged releases :)

Comment (kubukoz) [reply src/main/scala/com/kubukoz/next/Main.scala:91]: Also, about the format, I would print this in the welcome message:

```
Loading REPL...
Welcome to the spotify-next ${BuildInfo.version} REPL! Type in a command to begin
```

This is a couple lines below.

File: project/plugins.sbt [hunk 2]
```diff
@@ -5,3 +5,6 @@ addSbtPlugin("com.typesafe.sbt" % "sbt-native-packager" % "1.8.1")
 addSbtPlugin("org.scala-js" % "sbt-scalajs" % "1.5.1")
 addSbtPlugin("ch.epfl.scala" % "sbt-scalajs-bundler" % "0.20.0")
 addSbtPlugin("ch.epfl.scala" % "sbt-scala3-migrate" % "0.3.3")
+
+addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.10.0")
+addSbtPlugin("com.dwijnand" % "sbt-dynver" % "4.1.1")
```
Comment (kubukoz) [inline project/plugins.sbt:10]: We already have sbt-dynver thanks to the sbt-ci-release dependency.

File: build.sbt [hunk 3]
```diff
@@ -106,10 +106,13 @@ val next =
         "io.circe" %% "circe-literal" % "0.14.0-M5",
         // yes macros
         "com.github.julien-truffaut" %% "monocle-macro" % "3.0.0-M4"
-      )
+      ),
+      buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion),
```
Comment (kubukoz) [inline build.sbt:110]: `version` alone is good enough for now

**Review by kubukoz — CHANGES_REQUESTED** [review]
Thanks! Just some small changes I'd like to see here.

**Comment by toshihidetagami** [comment]
Thanks for the review! I added a commit.

**Comment by toshihidetagami** [comment]
Thanks for the review! I added commits.

**Review by kubukoz — APPROVED** [review]
Thank you!

**Comment by kubukoz** [comment]
Rebased and merged locally.

**Pull request closed without merging**